readme.md

```diff
@@ -162,13 +227,13 @@ Your Rails app has the following `application.html.erb`. Nothing shows up in you
 </html>
 ```
 
-> Your answer...
+Clark Answer </html>
 
```

Comment:
True, but the more important part here is that `yield` is missing an `=`. So we want...

```rb
<%= yield %>
```